backend/Agent/research_paper/server/parlant_nlp_adapter.py

```
@@ -0,0 +1,345 @@
+        generator.__orig_class__ = HealthcareSchematicGenerator[t]  # type: ignore
+        return generator  # type: ignore
+
+    async def get_embedder(self) -> BaseEmbedder:
+        """Return the embedder"""
+        return HealthcareEmbedder(
+            healthcare_service=self._healthcare_service,
+            logger=self._logger,
+            meter=self._meter,
+        )
+
+    async def get_moderation_service(self) -> ModerationService:
```

Ruff (0.14.5)

Undefined name `ModerationService`

(F821)